plugins/amazonq/mynah-ui/src/mynah-ui/ui/tabs/generator.ts

```diff
@@ -76,6 +76,17 @@ I can help you upgrade your Java 8 and 11 codebases to Java 17.
                 'Use of Amazon Q is subject to the [AWS Responsible AI Policy](https://aws.amazon.com/machine-learning/responsible-ai/policy/).',
             quickActionCommands: this.quickActionsGenerator.generateForTab(tabType),
             promptInputPlaceholder: this.tabInputPlaceholder.get(tabType),
+            contextCommands: [
+                {
+                    groupName: 'Metion code',
+                    commands: [
+                        {
+                            command: '@workspace',
+                            description: '(BETA) Reference all code in workspace.',
+                        },
+                    ],
+                },
+            ],
```

Comment:
need more context on why we can't put this into quickActionCommands as /dev, /transform

Reply:
I think it was a UX decision